apps/code-examples/src/app/code-examples/datetime/date-range-picker/basic/demo.component.ts

```diff
@@ -36,26 +39,26 @@ function dateRangeExcludesWeekend(
   imports: [FormsModule, ReactiveFormsModule, SkyDateRangePickerModule],
 })
 export class DemoComponent {
-  protected get reactiveRange(): AbstractControl | null {
-    return this.formGroup.get('lastDonation');
-  }
-
   protected dateFormat: string | undefined;
   protected disabled = false;
-  protected formGroup: FormGroup;
   protected hintText =
     'Donations received today are updated at the top of each hour.';
   protected labelText = 'Last donation';
-  protected lastDonation: FormControl<string | null>;
-  protected required = true;
 
-  constructor() {
-    this.lastDonation = new FormControl<string | null>(
-      { value: '', disabled: this.disabled },
-      [dateRangeExcludesWeekend],
-    );
-    this.formGroup = inject(FormBuilder).group({
-      lastDonation: this.lastDonation,
-    });
-  }
+  protected lastDonation = new FormControl<SkyDateRangeCalculation>(
+    {
+      value: {
+        calculatorId: SkyDateRangeCalculatorId.AnyTime,
```

Comment:
This is a more appropriate initial value for the date range picker (not an empty string).